vms/home/templates/home/home.html

```diff
@@ -31,4 +31,4 @@ <h1 class="index">Volunteer Management System</h1>
     </div>
     {% endif %}
 </div>
-{% endblock %}
\ No newline at end of file
+{% endblock %}
```

Comment:
What did you change? It's same as before. You were supposed to add a new line as suggested.

Reply:
I just need to add the `<br>` tag if I'm not wrong?

Reply:
The warning red circle minus sign (in the previous commit) went, that was all that was needed, it was added, won't be evident in the commit, I guess. Like [this file](https://github.com/anitab-org/vms/blob/develop/vms/home/templates/home/home.html), we won't see in commit message, but on trying to edit, newline shows at the end.

Reply:
Not a <br> tag. Just an Enter. According to the code quality, one must leave an empty line at the end of the file.

Reply:
I did add a new line now but I'm unable to see it when I open the file.

Reply:
As long as the warning, "No newline at end of file" is not there, it's good, it's already added and it's fine if you're unable to see.